x-pack/solutions/security/packages/ecs_data_quality_dashboard/impl/data_quality_panel/mock/test_providers/test_providers.tsx

```diff
@@ -66,30 +67,32 @@ const TestExternalProvidersComponent: React.FC<TestExternalProvidersProps> = ({
   });
 
   return (
-    <I18nProvider>
-      <ThemeProvider theme={() => ({ eui: euiDarkVars, darkMode: true })}>
-        <QueryClientProvider client={queryClient}>
-          <AssistantProvider
-            actionTypeRegistry={actionTypeRegistry}
-            assistantAvailability={mockAssistantAvailability}
-            augmentMessageCodeBlocks={jest.fn()}
-            basePath={'https://localhost:5601/kbn'}
-            docLinks={{
-              ELASTIC_WEBSITE_URL: 'https://www.elastic.co/',
-              DOC_LINK_VERSION: 'current',
-            }}
-            getComments={mockGetComments}
-            http={mockHttp}
-            baseConversations={{}}
-            navigateToApp={mockNavigateToApp}
-            currentAppId={'securitySolutionUI'}
-            userProfileService={jest.fn() as unknown as UserProfileService}
-          >
-            {children}
-          </AssistantProvider>
-        </QueryClientProvider>
-      </ThemeProvider>
-    </I18nProvider>
+    <KibanaRenderContextProvider {...coreMock.createStart()}>
```

Comment:
Usually there is no need for `I18nProvider` and `EuiThemeProvider` if `KibanaRenderContextProvider` is present, but I am keeping those here for completeness because that's exactly the transitive setup we use in security_solution with both existing to better simulate the environment. But since this package is supposed to be shared we can later remove those and only leave `KibanaRenderContextProvider`